### DISCUSSION TITLE
FanControl Hysteresis for GPU

### DISCUSSION THREAD

**Discussion by Allomaternal** [Rem0o/FanControl.Releases#1752] [Q&A]
Hello! I am trying to figure out how to make hysteresis work. I'm sure this question gets asked often and I have looked at other posts regarding this but answers never really seemed clear to me. I have a graph curve set up with 4 points that will leave my GPU fans on 0% until reaching 55c which will then turn on both fans to 35%. I set a Hysteresis of 10c with 3 second response time and checked for only on the way down. Maybe I am misunderstanding how it should be working but I had thought this meant that the fans will only drop back down to 0% once reaching 45c (55c - 10c = 45c). As soon as the temperature drops from the 55c point (which is certain to happen immediately) the fans drop down to 0% rather than taking into account the hysteresis setting. Am I misunderstanding how it works? I will leave an image of my current graph curve. Thank you very much! 

https://imgur.com/a/aI3ybiG

**Comment by Rem0o** [comment]
There a 2 things:

1) Hysteresis will be applied from the last point a change occurred. It may not be 45C.
2) In a graph, hysteresis is ignored at the limits, aka the most left and right point you set. If you want hysteresis to be applied pass these points, simply add duplicate points at like 30 degree and 90 degree.
